src/WebJobs.Script/Host/ScriptHost.cs

```diff
@@ -1001,8 +1002,8 @@ private Collection<FunctionMetadata> LoadProxyRoutes(string proxiesJson)
             {
                 try
                 {
-                    // Proxy names should follow the same naming restrictions as in function names.
-                    ValidateName(route.Name, true);
+                    // Proxy names should follow the same naming restrictions as in function names. If not, invalid characters will be removed.
```

Comment:
This doesn't seem related to the hotfix - can you explain this?

Reply:
This is part2 of the hotfix, I sent an email about it last night. many customers are hitting it as well.

---
In reply to: [146596874](https://github.com/Azure/azure-webjobs-sdk-script/pull/2060#discussion_r146596874) [](ancestors = 146596874)